src/wagtail_ai/prompts.py

```diff
@@ -1,17 +1,17 @@
 import inspect
 from dataclasses import dataclass
-from enum import Enum
 from typing import Union
 
-from django.conf import settings
 from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 
 @dataclass(kw_only=True)
 class Prompt:
```

Comment:
**Question:** Do we need this dataclass anymore now we only use the database prompts?